CONTRIBUTING.md

```diff
@@ -90,6 +90,22 @@
       `void function(const size_t /* count */) { ... }`). If the context does not provide a good name (such as
       `/* count */`), use `/* unused */`.
 
+* In general, we declare (member) variables of the same type in an own line each ([isolated declaration](https://clang.llvm.org/extra/clang-tidy/checks/readability/isolate-declaration.html)):
+  ```c++
+  class Foo {
+    ...
+    int a;
+    int b;
+  };
+  ```
+  However, certain test classes have many member variables of the same type, e.g., `std::shared_ptr<LQPColumnExpression>` for LQP node tests. Thus, we permit violations of isolated declaration in such test cases if the number of variables is high (see below). Please use common sense to decide on which format you use.
```

Comment:
> Thus, we permit violations of isolated declaration in such test cases if the number of variables is high (see below).

I would change this legislative tone. Addressing the reader, something like the following might work:
```
Declarations with more than one variables can be used in such test cases if [...]
```